tests/charts/nm_vllm_grafana_dashboards/test_config_map_list.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict, Optional
+
+from pytest_helm_templates import HelmRunner
+
+
+def test_expected_dashboard_config_maps_are_included(
```

Comment:
@DaltheCow in this test or another, can you test that at least the `label` label and it's respective value is correct? I think there are other labels to probably test too. You could likely steal some tests from the `nm-vllm` chart tests since I included tests for a lot of the same labels there.